wordpress-mobile/WordPress-iOS
# Add new devices icon for web preview

Adds the new [materials icon](https://material.io/resources/icons/?search=devices&icon=devices&style=baseline) for preview devices.

| Before | After |
|:--:|:--:|
| ![iOS 13 - Portrait](https://user-images.githubusercontent.com/3250/72851530-2482b580-3c69-11ea-8cf0-b069c1087258.png) | ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-01-21 at 15 59 33](https://user-images.githubusercontent.com/3250/72851571-3c5a3980-3c69-11ea-9e57-c84608a19c18.png) |

To test:

- Open blog or page preview and check icon in bottom right.

PR submission checklist:

- [x] ~I have considered adding unit tests where possible.~ Small icon change

- [x] ~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~ Not released yet.
